com.unity.render-pipelines.high-definition/Runtime/Material/VTBufferManager.cs

```diff
@@ -0,0 +1,110 @@
+#if ENABLE_VIRTUALTEXTURES
+using VirtualTexturing = UnityEngine.Rendering.VirtualTexturing;
+using UnityEngine.Experimental.Rendering;
+
+namespace  UnityEngine.Rendering.HighDefinition
+{
+    public class VTBufferManager
+    {
+        public static GraphicsFormat GetFeedbackBufferFormat()
+        {
+            return GraphicsFormat.R8G8B8A8_UNorm;
+        }
+
+        RTHandle m_VTFeedbackBuffer;
+        VirtualTexturing.Resolver m_Resolver = new VirtualTexturing.Resolver();
+        public static int AdditionalForwardRT = 1;
+        const int resolveScaleFactor = 16;
+        Vector2 resolverScale = new Vector2(1.0f / (float)resolveScaleFactor, 1.0f / (float)resolveScaleFactor);
+        RTHandle lowresResolver;
+        ComputeShader downSampleCS;
+
+        public VTBufferManager(HDRenderPipelineAsset asset)
+        {
+            downSampleCS = asset.renderPipelineResources.shaders.VTFeedbackDownsample;
+        }
+
+        public void CreateBuffers(RenderPipelineSettings settings)
+        {
+            if (settings.supportMSAA || settings.supportedLitShaderMode == RenderPipelineSettings.SupportedLitShaderMode.ForwardOnly)
+            {
+                // Our processing handles both MSAA and regular buffers so we don't need to explicitly resolve here saving a buffer
```

Comment:
question: have you tested this PR with MSAA? (@TomasKiniulis  for some coverage with MSAA)

Reply:
Done (and bugs fixed ;-) )